app/views/abuse_reports/new.html.haml

```diff
@@ -1,6 +1,6 @@
 - page_title "Report abuse"
 %h3.page-title Report abuse
-%p Please use this form to report users who create spam issues or comments or who otherwise behave inappropriately.
+%p Please use this form to report users who create spam issues, comments or behave inappropriately.
```

Comment:
Just for me (as a non native english speaker): The comma between `comments` and `or` is optional, isn't it?